interfaces/apparmor/backend_test.go

```diff
@@ -366,6 +370,9 @@ snippet
 }}
 
 func (s *backendSuite) TestCombineSnippets(c *C) {
+	restore := release.MockAppArmorSupportLevel(aa.FullSupport)
+	defer restore()
+
 	// NOTE: replace the real template with a shorter variant
 	restoreTemplate := apparmor.MockTemplate("\n" +
```

Comment:
The mixture of 'aa.Foo' and 'apparmor.Foo' here and elsewhere in the PR seems weird to me. Is there a reason why you aren't consistently using one over the other?